src/Microcosm.js

```diff
@@ -73,20 +74,17 @@ class Microcosm extends Foliage {
     return this
   }
 
-  prepare(action, ...buffer) {
+  prepare(/* action, ...buffer */) {
```

Comment:
It's silly, but the compiled code is smaller. I'll run the numbers, the confusion is not worth 20 bytes